runtime/config/config_test.go

```diff
@@ -163,9 +163,8 @@ var _ = Describe("Config", func() {
 				Expect(err).To(BeAssignableToTypeOf(&config.BundleConfigValidationError{}))
 				Expect(err.Error()).To(ContainSubstring(fmt.Sprintf("bundle config.json is invalid: %s:", bundlePath)))
 				Expect(err.Error()).To(ContainSubstring("'windows' MUST be set when platform is `windows`"))
-				Expect(err.Error()).To(ContainSubstring(`'Spec.Version' should not be empty.`))
-				Expect(err.Error()).To(ContainSubstring(`'Process.Args' should not be empty.`))
```

Comment:
Note: Process.Args was made an optional field in its struct in the upstream code definition since the last version of the package we pulled in. The upstream validation logic looks at fields that do not have `json:omitempty` tags, which Process.Args now has.